all.json

```
@@ -28,6 +28,7 @@
 	],
 	"deny": [
 		"07e96d43-381a-46a3-9c16-6daf97213efc.co",
+		"synchroconnects.mgmart.shop",
```

```suggestion
		"synchroconnects.mgmart.shop",
		"mgmart.shop",
```

Adding the top-level, no content.